src/parsy/__init__.py

```diff
@@ -332,6 +332,29 @@ def regex_parser(stream, index):
     return regex_parser
 
 
+def test(func, description):
+    @Parser
+    def test_parser(stream, index):
+        if index < len(stream):
+            char = stream[index:index + 1]
+            if func(char):
+                return Result.success(index + 1, char)
+        return Result.failure(index, description)
+
+    test_parser.__name__ = 'test_parser<%s>' % func.__name__
+
+    return test_parser
+
+
+def string_from(*strings):
+    # Sort longest first, so that backtracking works correctly
+    return alt(*map(string, sorted(strings, key=lambda s: -len(s))))
```

Comment:
As I mentioned before, use `reverse` argument instead of negating `len`, it's much cleaner.